tests/test_schedules.py

```diff
@@ -6,6 +6,21 @@
 from SYS_ATL.libs.memories import GEMM_SCRATCH
 from .helper import TMP_DIR, generate_lib
 
+
+def test_rearrange_dim():
+    @proc
+    def foo(N : size, M : size, K : size, x : i8[N, M, K]):
+        a : i8[N,M,K]
```

Comment:
Another point on formatting: `[N, M, K]` or `[N,M,K]`?

Reply:
pretty printing is `N, M, K`, so the former is more consistent?